packages/turbo-workspaces/src/updateDependencies.ts

```diff
@@ -37,6 +37,10 @@ function updateDependencyList({
   return { dependencyList, updated };
 }
 
+/**
+ * Convert workspace dependencies to the format that `to` requires. Only needed when pnpm is involved as
+ * it requires `workspace:*` and all the rest support `*`
```

Comment:
Did you characterize the edge cases for `bun`? Can you enumerate them?

Reply:
Most of them seem to resolve around not starting with a clean project (no node_modules anywhere)